docs/event-logging.rst

```diff
@@ -30,11 +31,11 @@ This example is taken from the `sample ERC20 contract <https://github.com/vyperl
        # All done, log the event for listeners
        log Transfer(msg.sender, _to, _value)
 
-Let's look at what this is doing. First, we declare two event types to log. The two events are similar in that they contain
-two indexed address fields. Indexed fields do not make up part of the event data itself, but can be searched by clients that
-want to catch the event. Also, each event contains one single data field, in each case called ``value``. Events can contain several arguments with any names desired.
+Let's look at what this is doing.
 
-Next, in the ``transfer`` function, after we do whatever work is necessary, we log the event. We pass three arguments, corresponding with the three arguments of the Transfer event declaration.
+    1. We declare two event types to log. The two events are similar in that they contain
+    two indexed address fields. Indexed fields do not make up part of the event data itself, but can be searched by clients that want to catch the event. Also, each event contains one single data field, in each case called ``value``. Events can contain several arguments with any names desired.
+    2. In the ``transfer`` function, after we do whatever work is necessary, we log the event. We pass three arguments, corresponding with the three arguments of the Transfer event declaration.
```

Comment:
Not actually a list

Reply:
Because of the numbering? I did that because the original content used "first" and "next".  It's not fully sequential, but you can't log an event without first declaring it.

Reply:
Oh I mean it literally isn't rendering as a list, if that was the intent

Reply:
oh weird.